docs/reference/cli-reference/cli-ref-push.md

```diff
@@ -30,7 +30,8 @@ where `<packagePath>` identifies the package to push to the server.
 
 - **`-ApiKey`**
 
-  The API key for the target repository. If not present,  the one specified in the config file is used.
+  The API key for the target repository. If not present,  the one specified in the config file is used. 
```

Comment:
Similar as the other feedback, we should prefer having the exact same description as the tooling. 

After all, AzDO is not the only source.